app/views/caseload/index.html.erb

```diff
@@ -61,12 +61,14 @@
         </a>
         <%= sort_arrow('last_name') %>
       </th>
+
       <th class="govuk-table__header" scope="col">
-        <a href="<%= sort_link('latest_movement_date') %>">
+        <a href="<%= sort_link('location') %>">
```

Comment:
If the `location` method is removed from `AllocatedOffender`, then this will need to become `cell_location`

```suggestion
        <a href="<%= sort_link('cell_location') %>">
```